charts/skyflo/templates/engine-configmap.yaml

```
@@ -17,3 +17,4 @@ data:
   JWT_ACCESS_TOKEN_EXPIRE_MINUTES: {{ .Values.engine.config.jwtAccessTokenExpireMinutes | quote }}
   JWT_REFRESH_TOKEN_EXPIRE_DAYS: {{ .Values.engine.config.jwtRefreshTokenExpireDays | quote }}
   LLM_MAX_ITERATIONS: {{ .Values.engine.config.llmMaxIterations | quote }}
+  CORS_ORIGINS: {{ .Values.engine.config.corsOrigins | quote }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)